### PR TITLE
Community meeting updates

### LINKED ISSUE #402: Community meetings page
- [ ] Searching for "ome community meeting" brings up the conference
 - [x] https://www.openmicroscopy.org/on-the-web/ is linked from the footer but not clearly mentioned from https://www.openmicroscopy.org/support/
 - [x] https://www.openmicroscopy.org/on-the-web/ links to the 2019 meeting notes. An index of them is likely needed.
 - [x] The zoom link is not accessible
 - [ ] Migrate pre-2017 notes from https://www-legacy.openmicroscopy.org/site/community/minutes/conference-calls/2017/2017-01-17-tuesday-team-meeting

### PR DESCRIPTION
**Pull request by joshmoore**
fix #402 by including more information about how to join meetings as well as making the minutes available directly from the webpage.

Screen Shot 2020-09-30 at 08 27 15

 - [x] 2020 notes
 - [x] other notes
 - [ ] review size of or replace on-the-web image
 - [ ] shrink size of /minutes text

### CODE REVIEW

**Comment by joshmoore** [comment]
see also: https://github.com/ome/www.openmicroscopy.org/pull/286/files

**Comment by joshmoore** [comment]
Pushed a few WIP commits which allow rendering a repository of markdown files (https://github.com/joshmoore/meeting-minutes/tree/master/2020) similar to announcements/.

**Comment by pwalczysko** [comment]
Wanted to build the ci for this, but seeing that https://ci.openmicroscopy.org/view/Mgmt/job/WWW-merge/ is currently disabled.  Any idea what the reason is ?

**Comment by manics** [comment]
@pwalczysko This should be handled by snoopy.

**Comment by sbesson** [comment]
Sorry, that's part of my backlog. I have decommissioned the CI master job in favor of https://merge-ci.openmicroscopy.org/jenkins/job/WEBSITE-push/ which is run daily but should achieve what you want i.e. deploy a staging website to https://github.com/ome/www.openmicroscopy.org/pull/403#issuecomment-700604202

**Comment by pwalczysko** [comment]
@sbesson @manics thanks for the update. So I assume that the result of the build will still be deployed on https://snoopycrimecop.github.io/www.openmicroscopy.org/support/ ?

I guess the reason I do not see the changes there are the failed travis checks here (the build is recent enough)

**Comment by manics** [comment]
@pwalczysko Yes. If necessary you can trigger a rebuild of the push job and override the parameters to disable the status check

**Comment by joshmoore** [comment]
Now green.

**Comment by pwalczysko** [comment]
The PR is excluded, that is why the snoopy pages do not show it.

Built it locally instead.

The links work, and the new item in the "support" section makes sense. I have couple of reformulation suggestions, will put them into the diff text. On the "on-the-web" page itself, I guess the image next to the Community meeting minutes is too big iand does not bring much. Suggest either make it smaller or remove it completely. The image also disproportionately claims too much space for the meeting minutes on that page, and pushes the github links below it out of the page.

**Comment by joshmoore** [comment]
> The PR is excluded, that is why the snoopy pages do not show it.

re-included.

> the image ... is too big

Agreed! Especially as we start having more content on this page. I also wonder if "on-the-web" is the best location for this content.

File: support/index.html [hunk 1]
```diff
@@ -81,6 +81,16 @@ <h4>OME Training</h4>
                     </div>
                 </div>
             </div>
+            <div class="small-12 medium-4 columns">
+                <div class="card card-support">
+                    <div class="card-section">
+                        <i class="border-red icon-red fa fa-link fa-2x"></i>
+                        <h4>On the web</h4>
+                        <p class="card-caption">Follow us on social media or listen in to a community meeting.</p>
```
Comment (pwalczysko) [inline support/index.html:89]: did not find the link to the zoom, maybe it is either ``read the minutes`` instead, or maybe this is WiP ?

Comment (joshmoore) [reply support/index.html:89]: Yes, this goes back to https://github.com/ome/www.openmicroscopy.org/issues/402 -- I assume we need to publish it but we should all agree. That _likely_ means activating the waiting room.

**Comment by pwalczysko** [comment]
One more comment - possibly it would be a good opportunity to change this here, although not changed in the present PR:
Screen Shot 2020-09-29 at 12 40 39

In the bioformats twitter link description, the above formulation sounds too unfriendly and unnecesarily technical/industrial. Maybe we could reformulate in the lines of  "See ... (this and that) ... to learn (or "for")... latest changes, updates etc."

File: support/index.html (hunk 1 above)
Comment (mtbc) [reply support/index.html:89]: Could ask in AOB today.

**Comment by pwalczysko** [comment]
@joshmoore 

> I also wonder if "on-the-web" is the best location for this content.

Actually, as it is named atm ("on-the-web", the meeting minutes are out of place, I agree. But I would suggest it would be enough to change the "on-the-web" wording for something like "OME Team presents itself in the outer world" (this suggestion of mine is of course too long, but maybe this could be cleverly shortened, and the minutes could easily accommodate with that

**Comment by joshmoore** [comment]
"outreach"? :wink:

**Comment by pwalczysko** [comment]
> "outreach"? 😉

:)
Maybe "Catch up with OME" (maybe longer version "Catch up with OME on..." ?

**Comment by manics** [comment]
Will the website need to be tagged and deployed after every meeting? Or is this a one-off?

**Comment by joshmoore** [comment]
For the moment, yes. That deployment will need automating so that when dependabot updates the submodule the website is immediately released.

**Comment by joshmoore** [comment]
Currently looking for a way to skip link checking for this subdirectory since a) there are a bunch of misses and b) I don't think we want the burden long term.

**Comment by joshmoore** [comment]
NB: it occurs to me while fixing https://github.com/ome/www.openmicroscopy.org/pull/403/commits/af506728fcfc4bb040e92e31ceb982a3114cc381 that the collections concept would allow us to template the workshop events quite nicely.

cc @sbesson @pwalczysko @jburel

**Comment by sbesson** [comment]
I could not find this staged and looking into the deployment, it looks like we are hitting https://docs.github.com/github/working-with-github-pages/troubleshooting-jekyll-build-errors-for-github-pages-sites#invalid-submodule.

**Comment by sbesson** [comment]
The last commit updates the .gitmodules to use HTTP and comply with the requirements of GitHub Pages when submodules are involved. With it included, this PR is now staged e.g. at https://snoopycrimecop.github.io/www.openmicroscopy.org/minutes/ as expected.

Collections look really nice. In addition to the workshops, they might also offer the solution for allowing to group/filter our growing announcements list by component.

**Comment by manics** [comment]
https://snoopycrimecop.github.io/www.openmicroscopy.org/minutes/2020/2020-09-29.html
image
The badge is broken

Regarding the deployment job, is anyone already working on it? If not could we sidestep the issue by hosting www.openmicroscopy.org on GitHub pages?

**Comment by joshmoore** [comment]
> The badge is broken

Hmmm.... may be that we have to strip those on publication. cc: @will-moore

File: on-the-web/index.html [hunk 2]
```diff
@@ -52,8 +52,12 @@ <h3 class="social-media-feature-section-headline"><i class="fa fa-twitter" aria-
             </div>
             <div class="marketing-site-content-section-block">
                 <h3 class="marketing-site-content-section-block-header">OME Team Meeting Minutes</h3>
```
Comment (pwalczysko) [inline on-the-web/index.html:54]:
```suggestion
                <h3 class="marketing-site-content-section-block-header">OME Team Meeting</h3>
```

File: support/index.html [hunk 3]
```diff
@@ -81,6 +81,16 @@ <h4>OME Training</h4>
                     </div>
                 </div>
             </div>
+            <div class="small-12 medium-4 columns">
+                <div class="card card-support">
+                    <div class="card-section">
+                        <i class="border-red icon-red fa fa-link fa-2x"></i>
+                        <h4>On the web</h4>
+                        <p class="card-caption">Follow us on social media or listen in to a community meeting. Zoom link on request via social media or <a href="https://forum.image.sc/g/ome">image.sc</a></p>
```
Comment (pwalczysko) [inline support/index.html:89]:
```suggestion
                        <p class="card-caption">Follow us on social media, read minutes of or listen in to an OME Team meeting. Zoom link on request via social media or <a href="https://forum.image.sc/g/ome">image.sc</a></p>
```

**Comment by pwalczysko** [comment]
Screen Shot 2020-09-30 at 12 30 29

the link to "last meeting" goes to https://github.com/ome/meeting-minutes - not useful, I guess this will not be fixed in this PR, but maybe we could drop this link from the template or somesuch (not sure how it gets into the mix in frist place).

The minutes itself look nice, only thing - for externals, the "meeting minutes "Agenda template" as first item on the list is not helpful. Maybe we need to balance the usefulness for external reader with our own workflow need on this point (possibly it is very useful for ourselves to have the template there...)

File: minutes/index.html [hunk 4]
```diff
@@ -0,0 +1,30 @@
+---
+layout: news
+title: Meeting Minutes
+filename: minutes
+main-blurb: Minutes from the weekly OME community meetings
```
Comment (pwalczysko) [inline minutes/index.html:5]:
```suggestion
main-blurb: Minutes from the weekly OME Team meetings
```

**Comment by pwalczysko** [comment]
I made suggestions in sense ``community meeting`` -> ``Team meeting`` but happy if we brand it differently now. If we go for ``community meeting``, we need to change it in all places, e.g. see below

Screen Shot 2020-09-30 at 12 43 07

File: support/index.html [hunk 5]
```diff
@@ -81,6 +81,16 @@ <h4>OME Training</h4>
                     </div>
                 </div>
             </div>
+            <div class="small-12 medium-4 columns">
+                <div class="card card-support">
+                    <div class="card-section">
+                        <i class="border-red icon-red fa fa-link fa-2x"></i>
+                        <h4>On the web</h4>
```
Comment (pwalczysko) [inline support/index.html:88]:
```suggestion
                        <h4>Catch up with OME</h4>
```

Comment (pwalczysko) [reply support/index.html:88]: Unifying suggestion

File: support/index.html (hunk 3 above)
Comment (pwalczysko) [reply support/index.html:89]: Deleting this suggestion, as it makes the text in the box too long and thus misformats the layout. Will push a commit from local instead.

File: support/index.html (hunk 5 above)
Comment (pwalczysko) [reply support/index.html:88]: Deleting this suggestion. Will push a commit from local instead.

**Comment by pwalczysko** [comment]
https://github.com/ome/www.openmicroscopy.org/pull/403/commits/3af5ae196525d8d81d93f7819f69351ed54de4fa renames the on-the-web to "Catch up with OME" in the icon and the button, and shortens the text there to keep formatting in check (too long text does nto fit with the other icons length above). imho, user can first click onto the meetings and there only get the info about how to join the zoom, the Support page is too high level to go into such details

**Comment by pwalczysko** [comment]
https://github.com/ome/www.openmicroscopy.org/pull/403/commits/218e28a32d448488d917cfb93872f9652fbc654f is trying to align the "Old Resources" icon with the rest - it was too short, and without any circular logo. Added a key logo and made the text shorter, so that the lack of a button is not so obvious. The icon is still too short, btu it is much better. No idea how to perfectly match it with the "Catch up with OME"

File: on-the-web/index.html (hunk 2 above)
Comment (joshmoore) [reply on-the-web/index.html:54]: I'll push a commit for "OME Community Meeting"

File: minutes/index.html (hunk 4 above)
Comment (joshmoore) [reply minutes/index.html:5]: I'll unify on "community meeting"

**Comment by joshmoore** [comment]
> https://github.com/ome/www.openmicroscopy.org/pull/403#issuecomment-701333504 the link to "last meeting" goes to https://github.com/ome/meeting-minutes - not useful, I guess this will not be fixed in this PR, but maybe we could drop this link from the template or somesuch (not sure how it gets into the mix in first place).

I leave that for discussion outside this PR. I assume the easiest thing will eventually arise. (Note: the previously link _also_ went to the full directory)

**Comment by joshmoore** [comment]
The meetings image is still pretty large, but otherwise, all updates pushed, @pwalczysko

**Comment by pwalczysko** [comment]
> The meetings image is still pretty large, 

I am on it now, ta

File: support/index.html [hunk 6]
```diff
@@ -84,11 +84,21 @@ <h4>OME Training</h4>
             <div class="small-12 medium-4 columns">
                 <div class="card card-support">
                     <div class="card-section">
+                        <i class="border-red icon-red fa fa-link fa-2x"></i>
+                        <h4>Catch up with OME</h4>
+                        <p class="card-caption">Follow us on social media, read minutes of or listen in to an OME Team meeting.</p>
```
Comment (pwalczysko) [inline support/index.html:89]: Still a lingering "Team meeting" here

Comment (joshmoore) [reply support/index.html:89]: Ah, it likely slipped in on the rebase. Thank you.

**Comment by joshmoore** [comment]
> I am on it now, ta

I was wondering if we might not just go for something like https://fontawesome.com/icons/user-friends?style=solid

```
<i class="fas fa-user-friends"></i>
```
Screen Shot 2020-09-30 at 17 15 30

**Comment by pwalczysko** [comment]
> <i class="fas fa-user-friends"></i>

Whatever I tried, I could not make that work - it just did nothing as written, whether inside a class or outside of it, even addition to the file https://github.com/ome/www.openmicroscopy.org/blob/master/css/font-awesome.css and also to https://github.com/ome/www.openmicroscopy.org/blob/master/css/font-awesome.min.css , it still simply did not show any icon. Only the icons which are already in the https://github.com/ome/www.openmicroscopy.org/blob/master/css/font-awesome.css seem to work, the one @joshmoore suggests is not there and unfortunately I do not see how to add it successfully. Let me know if you know better.

Btw, https://github.com/ome/www.openmicroscopy.org/pull/403/commits/3d210b8fe281def306b330ad10ef15e2935652dd is a fairly decent try, even with the "single user in circle icon". The strange space between the icons and the text in the Github section of the on-the-web page is not introduced by my commit, but as the mega-image of the community meeting now vanished, it got a bit exposed - again, could not see how to get rid of it.

**Comment by pwalczysko** [comment]
built on https://snoopycrimecop.github.io/www.openmicroscopy.org/on-the-web/

**Comment by joshmoore** [comment]
Looks good to me!
Screen Shot 2020-09-30 at 19 12 26
I'd argue that we likely have our logo on their more often than is strictly necessary, but let's deal with that elsewhere.

Shall we make a last call for comments?

**Comment by pwalczysko** [comment]
> Shall we make a last call for comments?

I would think so.

**Review by sbesson — APPROVED** [review]

**Pull request merged**